addons/account/models/account_full_reconcile.py

```diff
@@ -27,9 +28,8 @@ def unlink(self):
         res = super().unlink()
 
         # Reverse all exchange moves at once.
-        today = fields.Date.context_today(self)
         default_values_list = [{
-            'date': today,
+            'date': date_utils.get_month(move._get_accounting_date(move.date, move._affect_tax_report()))[1],
```

Comment:
I think that's a bit obscure.
We should put that in a variable and name it clearly so it's more obvious to the readers